ui/src/app/components/resource-card.tsx

```diff
@@ -506,16 +506,18 @@ export class ResourceCard extends React.Component<Props, State> {
                         data-test-id='card'>
         <div style={{display: 'flex', flexDirection: 'column', alignItems: 'flex-start'}}>
           <div style={{display: 'flex', flexDirection: 'row', alignItems: 'flex-start'}}>
-            <ResourceCardMenu disabled={this.actionsDisabled}
+            <ResourceCardMenu disabled={false}
                               resourceType={this.resourceType}
                               onCloneResource={() => this.cloneResource()}
                               onCopyConceptSet={() => this.setState({copyingConceptSet: true})}
+                              canDelete={this.ownerPermission}
                               onDeleteResource={() => this.openConfirmDelete()}
```

Comment:
nit/question: in Javascript, can you just pass a function name for a no-arg method where a lambda is expected? I.e. can you just do 
`onDeleteResource={this.openConfirmDelete}`?

Reply:
Yes, you can do this. I suspect the reason it hasn't been done in the other cases is that Javascript functions are first class objects, so it's easy to screw up:
`onDeleteResource={() => this.openConfirmDelete}` would return the function `this.openConfirmDelete` rather than calling it.
See the `MenuItem` in concept-set-details for an example.
`onDeleteResource={this.openConfirmDelete()}` probably also does something weird, but I don't know offhand what it would do.

Reply:
You can, but it doesn't maintain `this` bindings. `() => this.method()` calls this.method within the current scope of this, whereas if you pass down the function, it will cal the function with `this` scoped to wherever you are.

There's another way to do it which is pass `this.method.bind(this)`, but in general that is less good, because IIRC it makes the function akin to a scoped global, as opposed to just providing a callback.